gulp-tasks/create-component.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Component creation.
+ */
+
+'use strict';
+
+const {dest, src} = require('gulp');
+const inquirer = require('inquirer');
+const options = require('../gulp-options');
+const rename = require('gulp-rename');
+const replace = require('gulp-replace');
+
+let nameQuestion = {
+  type: 'input',
+  name: 'component_name',
+  message: 'Please enter component name'
+};
+
+let typeQuestion = {
+  type: 'list',
+  name: 'component_type',
+  message: 'What type of component?',
+  choices: ['Atom', 'Molecule', 'Organism']
+};
+
+function checkIfComponentNotExists() {
+  // TODO: Add validation and checking existing components.
```

Comment:
This function doesn't work. If component exist and you'll to create the same one, you'll get an empty file from the template.

Reply:
that's why TODO here:)